macOS/UnitTests/Subscription/Mocks/MockSubscriptionFeatureAvailability.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MockSubscriptionFeatureAvailability.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import BrowserServicesKit
+
+class MockSubscriptionFeatureAvailability: SubscriptionFeatureAvailability {
```

Comment:
This is a duplicate of iOS / SubscriptionFeatureAvailabilityMock
If required in both iOS and mac, can you please remove this and move the iOS one in BSK SubscriptionTestingUtilities so can be shared?